Mathlib/Algebra/Category/AlgebraCat/Monoidal.lean

```diff
@@ -55,43 +55,14 @@ instance : MonoidalCategoryStruct (AlgebraCat.{u} R) where
   leftUnitor X := (Algebra.TensorProduct.lid R X).toAlgebraIso
   rightUnitor X := (Algebra.TensorProduct.rid R R X).toAlgebraIso
 
-theorem forget₂_map_associator_hom (X Y Z : AlgebraCat.{u} R) :
-    (forget₂ (AlgebraCat R) (ModuleCat R)).map (α_ X Y Z).hom =
-      (α_
-        (forget₂ _ (ModuleCat R) |>.obj X)
-        (forget₂ _ (ModuleCat R) |>.obj Y)
-        (forget₂ _ (ModuleCat R) |>.obj Z)).hom := by
-  rfl
-
-theorem forget₂_map_associator_inv (X Y Z : AlgebraCat.{u} R) :
-    (forget₂ (AlgebraCat R) (ModuleCat R)).map (α_ X Y Z).inv =
-      (α_
-        (forget₂ _ (ModuleCat R) |>.obj X)
-        (forget₂ _ (ModuleCat R) |>.obj Y)
-        (forget₂ _ (ModuleCat R) |>.obj Z)).inv := by
-  rfl
```

Comment:
Were these lemmas themselves bad? I think when I wrote them, I wanted to characterize how `forget₂ (AlgebraCat R) (ModuleCat R)` commutes with the associator. Is there a way of recovering this elsewhere in the monoidal API, by bundling forget as a monoidal functor?

Reply:
The "more type correct" lemmas are those added in the companion PR [#16778](https://github.com/leanprover-community/mathlib4/pull/16778).

Reply:
Thanks! I hadn't expected the complexity that ends up there, but I guess nor am I surprised by it.

Do you know if we have the bundled `forget₂ (AlgebraCat R) (ModuleCat R)` as a `MonoidalFunctor`?

Reply:
When transporting the associator, we need to insert `F.μ : F.obj X ⊗ F.obj Y ⟶ F.obj (X ⊗ Y)` appropriate way in general. But for `forget₂` we can ignore it thanks to the defeq property. But it turned out that this defeq was very heavy.

Reply:
> Do you know if we have the bundled forget₂ (AlgebraCat R) (ModuleCat R) as a MonoidalFunctor?

Yes, it's defined at the bottom of this file (`toModuleCatMonoidalFunctor`).